submissions/bukkyOyetimehin/challenge1-arrays/question1.js

```diff
@@ -0,0 +1,28 @@
+var longestCommonPrefix = function (strs) {
```

Comment:
Excellent. Welldone.
Concise and really good time and space complexities too.

Something I noticed though, in your time complexity, M would actually be the length of the shortest string.